index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Tic Tac Toe</title>
+	<link href='https://fonts.googleapis.com/css?family=Noto+Sans:400,700,700italic' rel='stylesheet' type='text/css'>
+	<link rel="stylesheet" type="text/css" href="css/style.css">
+
+</head>
+<body>
+	<div id="header">
+		<h1>Tic Tac Toe!</h1>
+		<div id="sub-header">
+			<span id="x-span">X</span>
+			<h2>Play if you dare.</h2>
+			<span id="o-span">O</span>
+		</div>
+	</div>
+
+	<div id="game-board">
+		<div id="row-1">
+			<div class="play-square" id="cell-1">
+			</div>
+			<div class="play-square" id="cell-2">
+			</div>
+			<div class="play-square" id="cell-3">
+			</div>
+		</div>
+		<div id="row-2">
+			<div class="play-square" id="cell-4">
+			</div>
+			<div class="play-square" id="cell-5">
+			</div>
+			<div class="play-square" id="cell-6">
+			</div>
+		</div>
+		<div id="row-3">
+			<div class="play-square" id="cell-7">
+			</div>
+			<div class="play-square" id="cell-8">
+			</div>
+			<div class="play-square" id="cell-9">
+			</div>
+		</div>
+	</div>
+
+	<div id="footer">
+		<button id="clear-button" class="footer-element">Clear!</button>
+		<p id="winner-name" class="footer-element">Who will survive?</p>
+		<button id="single-player" class="footer-element">Vs. Comp</button>
+	</div>
+	<script src="js/script.js"></script>
+</body>
+</html>
+
+<!-- why do div boxes collapse when placed inline? this is fixed with inline-block, but still...?
+     why do the boxes change possitions when p-tags and text are placed in them?
+     getElementsByClassName("whatever").addEventListener() isn't valid? What's up with that?
+     I made an event listener for each cell id, but there must be a more efficient way, right?
+     -->
```

Comment:
Good question. Inline is kinda weird because it ignores the width that you give the element, and defaults to only being large enough to fill the content inside the element. You'll notice this if you set the `.play-square` elements to `inline`, they collapse. But try setting them to `inline` and placing text inside the divs. Sure enough, they now have width, but only enough to fit the content.

Another weird workaround to this is to set the `.play-square` elements to `inline` and `float: left`. Then the elements don't collapse, but they're taken out of the flow of the page.